GNUmakefile

```diff
@@ -10,7 +10,7 @@ endif
 PKG_NAME=pkg/artifactory
 # if this path ever changes, you need to also update the 'ldflags' value in .goreleaser.yml
 PKG_VERSION_PATH=github.com/jfrog/terraform-provider-${PRODUCT}/${PKG_NAME}
-PROVIDER_VERSION?=$(shell git tag --sort=-creatordate | head -1 | sed  -n 's/v\([0-9]*\).\([0-9]*\).\([0-9]*\)/\1.\2.\3/p')
+PROVIDER_VERSION?=$(shell git tag --sort=-creatordate  | grep v7 | head -1 | sed  -n 's/v\([0-9]*\).\([0-9]*\).\([0-9]*\)/\1.\2.\3/p')
```

Comment:
In my current branch, I use `git describe --tags --abbrev=0` instead which will get the latest tag that belongs to the parent branch. So no need for `grep` or `head -1` as well.

Reply:
Thanks, I'll use it, too.